karma.conf.js

```diff
@@ -24,7 +24,8 @@ module.exports = function(config) {
 			// source files, that you wanna generate coverage for
 			// do not include tests or libraries
 			// (these files will be instrumented by Istanbul)
-			'app/scripts/**/*.js': sourcePreprocessors
+			'app/scripts/**/*.js': sourcePreprocessors,
+			'test/spec/**/*.js': sourcePreprocessors
```

Comment:
Sorry I didn't pay attention to the comments.

It is an long discussion to include unit tests in coverage or not.
I prefer to include them as they ensure that all lines of test code are executed, I mean, would be possible to get some line of expected assertion never been called and without coverage you never will be able to find out and the will never be evaluated.

Regarding messing values, while it would be true I managed to get a 100% test case coverage which should not affect too much on library values apart from more lines counted for percentage.

I would let them in but let me know what you think and will follow...